src/fmeval/eval_algorithms/summarization_accuracy.py

```diff
@@ -150,74 +172,37 @@ def evaluate(
         model: Optional[ModelRunner] = None,
         dataset_config: Optional[DataConfig] = None,
         prompt_template: Optional[str] = None,
+        num_records: int = 100,
         save: bool = False,
-        num_records=100,
     ) -> List[EvalOutput]:
-        """Perform summarization accuracy evaluation on a dataset.
+        """Compute summarization accuracy metrics on one or more datasets.
```

Comment:
NTH if this is for more than one dataset, shouldn't we update the signature to take optionally both a dataset config or a sequence thereof?

Reply:
Not quite. Currently, we only perform evaluations on multiple datasets if `dataset_config` is `None`, in which case we use all of the built-in datasets. Otherwise, if the user provides their own dataset (via a single `DataConfig`), we run the evaluation on that single dataset.

Reply:
yeah indeed. this doesn't look very clean to me.. but ok let's keep how it is right now since we don't want to change the external api